### DISCUSSION TITLE
can't see logCampaignDetails event in firebase debug view

### DISCUSSION THREAD

**Discussion by leandro-faria** [invertase/react-native-firebase#6378] [Q&A]
I've been trying to send logCampaingDetails using the react-native-firebase, but I can't see the events arriving at debug view. Even if I try to send a custom event with `campaing_details`, I still can't see them.

My goal is to send this event when receiving a push notification, like this:

```TS
import PushNotification, { ReceivedNotification } from 'react-native-push-notification';
import rnanalytics, { FirebaseAnalyticsTypes } from '@react-native-firebase/analytics';
[...]
    PushNotification.configure({
      onNotification: (notification: ReceivedNotification) => {
        rnanalytics().logCampaignDetails({ campaign: 'utmTestCampaign', source: 'utmTestSource', medium: 'utmTestMedium' });
```

This event won't appear at debug view, and the next events (custom events, screen view, etc) are not associated to the utm I send

Is this something happening only in debug view, or should it appear there?

**Comment by leandro-faria** [comment]
`"@react-native-firebase/analytics": "12.8.0"`

**Reply by mikehardy** [reply]
Debugging code targeted for mobile from nearly a year ago is not on the list of priorities 😰 you know how fast it changes, and the main job here is to keep pace with the underlying SDK releases (which we do) since those fix bugs like this. Please update to current versions prior to logging issues to make sure we're not chasing things that are already fixed. Can you verify this is still happening with current versions?

Additionally you should enable analytics debugging https://firebase.google.com/docs/analytics/debugview#android and watch adb logcat - it may help

**Comment by PabloMD** [comment]
`@react-native-firebase/analytics@^17.3.0"` - @mikehardy  I'm not able to see the events in Debug mode.... 

@leandro-faria does it work in general and the issue is related only to DebugView functionality?
